training-event/src/main/java/org/example/TrainingEventApplication.java

```diff
@@ -1,4 +1,4 @@
-package org.example.trainingevent;
+package org.example;
```

Comment:
each service should have its own package. Avoid generic naming, it's good thing to use corporate prefix, but still each service/library will have own sub package.
`org.example.trainingevent` and `org.example.gym`
Or even `org.example.trainings.statistics` and `org.example.trainings.gymmanagmenet` - preserving common domain, but providing more context